recipe/conda_build_config.yaml

```diff
@@ -11,17 +11,18 @@ channel_targets:
   - tiledb main
 channel_priority:
   - strict
-# conda-forge has migrated to fmt 10 and spdlog 1.12. We need fmt 9 to install
+# conda-forge has migrated to fmt 11 and spdlog 1.14. We need fmt 9 to install
```

Comment:
> We need fmt 9 to install

Is this (still) true? I don't see any pins in the image files for fmt or spdlog, and the feedstock-for-cloud version of the tiledb package doesn't depend on them.